README.md

```diff
@@ -9,36 +9,30 @@ The premier gallery experience for your NFTs.
 1. [Install Yarn](https://classic.yarnpkg.com/en/docs/install)
 2. [Install NVM](https://github.com/nvm-sh/nvm)
 
-### Setup
-
-Create a local .env file by copying values from .env.sample:
-
-```terminal
-$ cp .env.sample .env
-```
+### Setup Web
 
 Install our node version:
 
 ```bash
-$ nvm use
+nvm install && nvm use
 ```
 
 Install dependencies:
 
 ```bash
-$ yarn
+yarn install
 ```
 
-Generate relay files:
+Create a local .env file by copying values from .env.sample:
 
 ```bash
-yarn relay
+cp apps/web/.env.sample apps/web/.env
 ```
 
 Start app:
 
 ```bash
-$ yarn dev
+yarn moon run web:dev
```

Comment:
here the script is `yarn moon`, but in the `other commands` section below it's just `moon`.

probably worth adding in this readme a recommendation to download moon, and that the docs will assume you have it enabled (and that `yarn` can be used as a backup)

Reply:
Updated the readme to talk more about this